cmd/git/README.md

```diff
@@ -0,0 +1,37 @@
+# Git Clone Wrapper
+
+**TL;DR:** As part of the build, the sources need to be retrieved. One option is to use `git` to clone the source to the container filesystem. This was used to be done by a Tekton Git Resource. This package contains Shipwright Build owned Git retrieval code, which is wrapping around the `git` CLI in a minimal container setup.
```

Comment:
The [image](https://github.com/shipwright-io/build/blob/master/images/git/Dockerfile) is handcrafted for this step, but I can add a quick note that if one wants to have their own image, what it would need to be working. The CLI code for the step checks for the binaries anyway, but we can make sure it is documented as well.

Reply:
would be good.